packages/now-cli/src/util/dev/server.ts

```diff
@@ -764,7 +803,40 @@ export default class DevServer {
     const { ig } = await getVercelIgnore(this.cwd);
     this.filter = ig.createFilter();
 
-    const nowConfig = await this.getNowConfig();
+    const devCommandPromise = this.runDevCommand();
+
+    let address: string | null = null;
+    while (typeof address !== 'string') {
```

Comment:
this moved up so `address` is available when `VERCEL_URL` is set.

Reply:
This also fixes a bug where if `VERCEL_URL` is set in your `.env` (like from `vc env pull`), it's stored and set to an empty string `VERCEL_URL=''`

<img width="410" alt="Screen Shot 2020-07-20 at 12 51 26 AM" src="https://user-images.githubusercontent.com/8675906/87913055-4ea00f00-ca23-11ea-9e01-f6632ecf4992.png">

Reply:
@TooTallNate